src/index.js

```diff
@@ -22,6 +24,17 @@ const forcedServices =
       }
     : {};
 
+// Initialize Google Analytics ID
+ReactGA.initialize('UA-532159-1');
+
+const history = createBrowserHistory();
+
+// Get the current location
+history.listen((location, action) => {
+	ReactGA.set({ page: location.pathname });
```

Comment:
Did you test it, for example with `console.log()` to verify that it is working?

Reply:
Yes, add `console.log()´ for test this function and this working.